app/lib/job_feed_importers/talroo_job_feed_importer.rb

```diff
@@ -34,8 +34,9 @@ def import
         end
       end
       req.run
-      ids = JobPosting.talroo.pluck(:id)
-      EnsureJobPostingsFullTextSearchJob.perform_later(ids)
+      JobPosting.talroo.pluck(:id).each_slice(1000) do |slice|
+        EnsureJobPostingsFullTextSearchJob.perform_later(slice)
+      end
```

Comment:
Not sure `each_slice` is needed since we're using `pluck` which loads all the ids into an array.